src/fondue/Fondue.js

```diff
@@ -405,11 +414,53 @@ export default class Fondue {
 		return text ? this._removeNullBytes(text) : null;
 	}
 
-	_removeNullBytes(value) {
-		// Currently Font.js returns null bytes in name table
-		// strings, we filter them here.
-		// https://github.com/Pomax/Font.js/issues/74
-		/* eslint-disable no-control-regex */
-		return value.replace(/\x00/g, "");
+	// Language support (from old Wakamai Fondue)
+	get languageSupport() {
+		const fontCharSet = this.supportedCharacters;
+		let result = [],
+			language,
+			chars,
+			charCode,
+			found,
+			i,
+			total,
+			ignoreList;
+
+		ignoreList = [
+			167, // section sign
+			8208, // hyphen
+			8224, // dagger
+			8225, // double dagger
+			8242, // prime
+			8243, // double prime
+			8274, // commercial minus sign
+			10216, // mathematical left angle bracket
+			10217, // mathematical right angle bracket
+		];
+
+		for (language in languageCharSets) {
+			chars = languageCharSets[language];
+			found = 0;
+			total = chars.length;
+
+			for (i = 0; i < total; i++) {
+				charCode = chars.codePointAt(i).toString(16);
+				if (
+					ignoreList.includes(charCode) ||
+					fontCharSet.includes(charCode)
+				) {
+					found += 1;
+				}
+			}
+
+			// We consider a language supported when *all* of the
+			// required characters are present in the font (except
+			// those in the ignoreList of course)
+			if (found === total) {
```

Comment:
Psst https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every ;-)

Note all that important in this case as there's not that much nesting going on.

Reply:
Ah, good to know, thanks! This functionality was wholesale copied from the old WF, and should be refactored, improved or completely replaced after we formed a plan [here](https://github.com/Wakamai-Fondue/wakamai-fondue-site/issues/29).